src/app/index.route.coffee

```diff
@@ -77,4 +84,19 @@ angular.module 'mnoEnterpriseAngular'
             controller: 'DashboardMarketplaceCompareCtrl'
             controllerAs: 'vm'
 
-    $urlRouterProvider.otherwise '/impac'
+    # $urlRouterProvider.otherwise '/impac'
+    
+    # TO DELETE
+    $urlRouterProvider.otherwise '/onboarding'
```

Comment:
TIP: `$urlRouterProvider.otherwise` argument can be a function returning the right state 😉